testplan/web_ui/testing/src/AssertionPane/AssertionTypes/basicAssertionUtils.js

```diff
@@ -3,6 +3,448 @@ import {hashCode} from '../../Common/utils';
 
 /** @module basicAssertionUtils */
 
+/**
+ * Content required to render a basic assertion.
+ * @typedef {Object} AssertionContent
+ * @property {object|string|null} preTitle - Content above the assertion title
+ * @property {object|string|null} preContent - Content between the preTitle and
+ *                                             the title
+ * @property {object|string|null} leftTitle - Left side of the title
+ * @property {object|string|null} rightTitle - Right side of the title
+ * @property {object|string|null} leftContent - Left side of the main content
+ * @property {object|string|null} rightContent - Right side of the main content
+ * @property {object|string|null} postTitle - Content immediately below the
```

Comment:
postTitle and postContent dont get used at all, I guess it's not a problem to keep them in anyway as they are only taking up 4 lines of code and could be used in the future - just a note

Reply:
Good point, I did think that there were perhaps a few too many options here...

I think removing those content sections would be a bigger change than a simple refactor, and require updating the BasicAssertion module too to not render those sections. Maybe a change we can make in future, though as you note it's not exactly doing much harm to leave them in...